### ISSUE TITLE
Use of populationName?

### ISSUE THREAD

**Issue by AnnikaRPS** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#541]
The "populationName" field of a PopulationSimulationSet is described as "display name of population" but I can't find the populationName that I defined in my workflow in the report. It seems that only SimulationSetNames are used in the report. Shouldn't the population name be used for example in the demography plot descriptions (maybe combined with the SimulationSetName)?

**Comment by Yuri05** [comment]
>Shouldn't the population name be used for example in the demography plot descriptions (maybe combined with the SimulationSetName)?

Agree. I also think for demography plots we should define sub-chapters in report (1 sub-chapter per simulation set)?
Then population name could be included into the name of subchapters, e.g. something like this
![grafik](https://user-images.githubusercontent.com/25061876/133416698-11775150-8c74-4e03-a165-e8493f3c6aab.png)

Additionally (or alternatively?) population names could be included into the figure captions
![grafik](https://user-images.githubusercontent.com/25061876/133417209-329d0509-8251-43d2-96ad-bab9b0f2735e.png)

**Comment by Yuri05** [comment]
@sfrechen @TWendl @Christoph27
